### ISSUE TITLE
Auto play last radio or first

### ISSUE THREAD

**Issue by puiuna** [NielsMasdorp/Nederadio-Android#43]
It's possible to autoplay the first radio station on the list or the last one played?

**Comment by NielsMasdorp** [comment]
Hi @puiuna, sorry I completely missed this issue. Apologies. You mean after a cold start?

**Comment by puiuna** [comment]
That's right, if the app is not in the background but has already been opened once in the past. Thank you for your attention.
